### ISSUE TITLE
Few Issues Found and Fixed

### ISSUE THREAD

**Issue by SAmundson07** [spliffz/X52Elite_v3.3.0-beyond#3]
Just wanted to shoot you a message to let you know about a few issues I found after updating to your latest build. I have been using the Version 2.3.3 since I got my HOTAS and love it, but I ran into a few minor issues with the files when updating to this newest build.

First one I noticed was that the Logitech profile that is suppose to be for the X52 Pro was blank aside from have the 6 mode profiles built out. Fortunately it looks like the profile that was intended to be for the standard X52 was still built for the X52Pro in the file so I used that instead, but still thought it was something worth pointing out.

Second issue was similar to something I ran into with the original build I started with, but slightly different. In the .bind file everything is tied to the device type "X52Pro" but needs to be tied to "SaitekX52Pro". It won't show up in the drop down menu in game unless that naming is changed. Again a minor issue if you know what to look for, but there are a lot of people who don't so I thought I would point it out. I attached the corrected .bind file if you want to replace the existing one.

[X52Elite v3.3.0-beyond-r1.3.0.zip](https://github.com/spliffz/X52Elite_v3.3.0-beyond/files/3847231/X52Elite.v3.3.0-beyond-r1.3.0.zip)

 

Thanks for putting the time into this build, I know personally what a time consuming pain it is to set all this up manually and find that right layout!

**Comment by obmucm** [comment]
Thank you so much, this is exactly the problem I had and I needed this fix.

**Comment by SAmundson07** [comment]
Glad to hear it. I had a feeling this might be the issue, but I'm happy to hear that solved you main issue. If you did end up using my updated profile and key binding and find any issues please let me know.

**Comment by obmucm** [comment]
I did find one issue so far.  I had to rebind the FSD Hyperspace as it wasn't bound correctly.

**Comment by SAmundson07** [comment]
Yeah I ran into several bindings and button profiles that weren't correct after I posted my first reply to you. You can find new profile and key binding here which has been fixed: https://github.com/SAmundson07/Elite-Dangerous

**Comment by obmucm** [comment]
Thanks a million.

I was trying to do it on the last profile you sent me, but was unable to.

I want to switch the pinkie modifier button with the Reverse thrust button, essentially just swap the two, but I can't quite get it to work in the profiler, any advice?

**Comment by SAmundson07** [comment]
Unfortunately you can't, that pinkie button is dedicated to that secondary feature so to my knowledge it doesn't allow you to utilize a different button for that function.

**Comment by LeeJacs** [comment]
> Yeah I ran into several bindings and button profiles that weren't correct after I posted my first reply to you. You can find new profile and key binding here which has been fixed: https://github.com/SAmundson07/Elite-Dangerous

Its saying that these files were updated 22days ago - is that correct.

Thanks for all your hard work!

**Comment by SAmundson07** [comment]
@LeeJacs Yes I uploaded the updated files around that time. As I was playing a friend of mine and myself continued to find issues with either the bindings or the profile so I decided to wait to update my file until we had some time to fix the majority of the issues. To my knowledge everything should be corrected with this set of files and everything has worked for me. Glad I can assist since I know how painful it can be to work with the X52 software and figure everything out.
